env.example

```diff
@@ -0,0 +1,24 @@
+# this is the externally available public path
+# that connects the client dev server to the correct
+# host, in case you have to override it
+BASE_PATH=/
+API_BASE_PATH=/
+STATIC_DIRECTORY=/static
+PUBLIC=https://twodotoh.a.influxcloud.dev.local/
+UI_SHA=local_dev_mode
+CLOUD_URL=/auth
+
+# NOTE: self signed certs aren't currently supported with
+# the gateway oidc provider plugin
+# PORT_HTTP=8080
+# PORT_HTTPS=8443
+PORT_HTTPS=8080
+PORT=8080
+
+INFLUXDB_ROOT=~/ui
```

Comment:
all of these are monitor-ci specific, not related to the running / building of this service

Reply:
Ya, I think I was confused by what was needed and what wasn't. I'll clean these out